NOTICE

```diff
@@ -0,0 +1,24 @@
+ 
+# This is the official list of copyright holders and authors.
+#
+# Often employers or academic institutions have ownership over code that is
+# written in certain circumstances, so please do due diligence to ensure that
+# you have the right to submit the code.
+#
+# When adding J Random Contributor's name to this file, either J's name on its
+# own or J's name associated with J's organization's name should be added,
+# depending on whether J's employer (or academic institution) has ownership
+# over code that is written for this project.
+#
+# How to add names to this file:
+#     Individual's name <submission email address>.
+#
+# If Individual's organization is copyright holder of her contributions add the
+# organization's name, optionally also the contributor's name:
+#
+#     Organization's name
+#         Individual's name <submission corporate email address>
+#
+# Please keep the list sorted.
+
+
```

Comment:
```suggestion
Darko Lukić <lukicdarkoo@gmail.com>
```
Or even add Cyberbotics Ltd. if this work has been conducted in the context of his employment.
// Cc @lukicdarkoo

Reply:
Thank you for bringing this up, it is a good point. I have just spoken to @omichel and we think it is a good idea to put `Cyberbotics Ltd.`.

Reply:
You have permission for pushing to this branch, feel free to add it and I'll merge.